public/js/load_comment.js

```diff
@@ -0,0 +1,104 @@
+
+//const & var
+const addComment = (data)=>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/soutenance14/ST/pullRequest?prid=8161867)
```suggestion
const addComment = (data) =>{
```